packages/remote-config/src/abt/experiment.ts

```diff
@@ -90,6 +87,7 @@ export class Experiment {
       const analytics = this.analyticsProvider.getImmediate({ optional: true });
       if (analytics) {
         analytics.setUserProperties({ properties: customProperty });
+        analytics.logEvent(`Received ABT experiment update`);
```

Comment:
Are event names allowed to have spaces? I think it might be safer to use "set_firebase_experiment_state" as Sam suggested.